cmd/webhook/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"k8s.io/component-base/logs"
 
 	"github.com/karmada-io/karmada/cmd/webhook/app"
+	_ "github.com/karmada-io/karmada/pkg/util/log"
```

Comment:
Is it also useful to import the package into a webhook?

Reply:
Roughly search in the code, find a log for webhook.
https://github.com/karmada-io/karmada/blob/0ec865ced6c8151c154c49c62883fcb12f839383/vendor/sigs.k8s.io/controller-runtime/pkg/manager/internal.go#L580-L582

Reply:
Thanks~